apps/services/discord/src/commands/dev/eval.ts

```
@@ -0,0 +1,102 @@
+        try {
+            if (code.includes("client.token") || code.match(/Token|bot_token|mongo/i)) {
+                evaled = "Haha, You Idiot nice try but I caught you!";
+            } else {
+                // Check if the code looks like TypeScript (ends with .ts or contains TypeScript-specific features)
+                if (code.includes('import') || code.includes('export')) {
+                    // Transpile the TypeScript code to JavaScript
+                    const tsCode = fromTS.transpileModule(code, {
+                        compilerOptions: { module: fromTS.ModuleKind.CommonJS },
+                    });
+                    // Execute the transpiled JavaScript code
+                    evaled = await eval(tsCode.outputText);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)